ddpui/tests/api_tests/test_dbt_api.py

```
@@ -285,14 +355,252 @@ def test_post_dbt_makedocs(
+
+
+def test_post_run_dbt_commands_with_payload(orguser: OrgUser, f_org_tasks):
+    """tests post_run_dbt_commands with TaskParameters payload"""
+    request = mock_request(orguser)
+    payload = TaskParameters(flags=["full-refresh"], options={"target": "dev"})
+
+    mock_task_id = "test-task-id-456"
+    mock_celery_task = Mock()
+    mock_celery_task.id = "celery-task-id"
+
+    with patch("ddpui.api.dbt_api.uuid4", return_value=mock_task_id) as mock_uuid, patch(
```

Ruff (0.11.9)

Local variable `mock_uuid` is assigned to but never used

Remove assignment to unused variable `mock_uuid`

(F841)